ui/v1/maslowWorker.js

```diff
@@ -46,19 +47,23 @@ const agent = async ({ ask, question }) => {
         const signature = '{ ' + Object.keys(api).join(', ') + ', ' + Object.keys(inputs).join(', ') + ' }';
         const foo = new Function(signature, values[0]);
         const returnVal = foo({ ...api, ...inputs });
-        return returnVal.toDisjointGeometry();
+        if (typeof returnVal === 'object') {
+          return returnVal.toDisjointGeometry();
+        } else {
+          return returnVal;
+        }
       case 'getLayoutSvgs':
         // Extract shapes
         let items = api.Shape.fromGeometry(values[0]).toItems();
```

Comment:
You might want to try unpacking a bit like this ...

case 'getLayoutSvgs': {
  const [g, margin, width, length] = values;
  ...
}